### PR TITLE
Delegate Provisioning refresh to Configuration

### PR DESCRIPTION
**Pull request by agrare**
This provider was missed during the changes for https://github.com/ManageIQ/manageiq/pull/20345 

Related: https://github.com/ManageIQ/manageiq/pull/22419

Dependent:
* https://github.com/ManageIQ/manageiq-schema/pull/685

### CODE REVIEW

**Comment by Fryguy** [comment]
Should this be petrosian/yes?

**Review by Fryguy — APPROVED** [review]
@jrafanie Would also like your review since this is reflected in https://github.com/ManageIQ/manageiq/pull/22419

File: config/settings.yml [hunk 1]
```diff
@@ -6,8 +6,6 @@
 :ems_refresh:
   :foreman_configuration:
     :refresh_interval: 15.minutes
-  :foreman_provisioning:
-    :refresh_interval: 1.hour
```
[the remark this answers is not in the document]
Comment (jrafanie) [reply config/settings.yml:10]: good question, I'm not sure what's looking for these possible values and if they'll gracefully handle old settings ... or we need to drop any changes from the settings.

File: app/models/manageiq/providers/foreman/configuration_manager/refresh_worker.rb [hunk 2]
```diff
@@ -4,16 +4,4 @@ class ManageIQ::Providers::Foreman::ConfigurationManager::RefreshWorker < MiqEms
...
-
-  # overriding queue_name_for_ems so PerEmsWorkerMixin picks up *all* of the
-  # Foreman-managers from here.
-  # This way, the refresher for Foreman's ConfigurationManager will refresh *all*
-  # of the inventory across all managers.
-  class << self
-    def queue_name_for_ems(ems)
-      return ems unless ems.kind_of?(ExtManagementSystem)
-
-      ems.provider.managers.collect(&:queue_name).sort
```
Comment (jrafanie) [inline app/models/manageiq/providers/foreman/configuration_manager/refresh_worker.rb:16]: `[]` => 🗑️ 🔥 🚽 

😄

**Comment by jrafanie** [comment]
LGTM, other than the question about needing a migration.

File: config/settings.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (agrare) [reply config/settings.yml:10]: Sure we can do that

[the remark this answers is not in the document]
Comment (agrare) [reply config/settings.yml:10]: https://github.com/ManageIQ/manageiq-schema/pull/685

**Comment by Fryguy** [comment]
Backported to `petrosian` in commit c875ee12066fbf2ca74671e75f79a2d187240a2e.

```text
commit c875ee12066fbf2ca74671e75f79a2d187240a2e
Author: Jason Frey <fryguy9@gmail.com>
Date:   Wed Mar 22 13:40:15 2023 -0400

    Merge pull request #113 from agrare/delegate_refresh_to_configuration_manager
    
    Delegate Provisioning refresh to Configuration
    
    (cherry picked from commit 91fb022320f12eb6289df6e4aa050251992fa1d3)
```

**Pull request merged**